practices/documentation_testing.md

```diff
@@ -0,0 +1,105 @@
+	- continuous delivery
+	- value
+	- good developer experience
+keywords:
+  - Developer experience
+  - Tech writing
+  - Getting started tutorial
+  - Testing your Documentation
+  - Tech writing feedback
+covered_by_dx_scanner: false
+---
+# Documentation Testing
+
+**TL;DR** Documentation testing is a process of improving your existing documentation through the Passive Feedback Gathering and Active Testing. Understanding how to work with feedback is crucial for building good documentation and positive developer experience.
+
+## What is Documentation Testing?
+
+Documentation testing is a few processes of continuous feedback gathering and documentation improvements based on feedback. You can imagine it as active listening to your developers. Similarly, as you use testing for your code, you should be testing your docs to provide clear and precise information that contributes to positive developer experience.
+
+Docs are a developer’s first impression of your product, yet they are often overlooked. Improving your Docs directly translate into improving first impressions. 
+
+![Documentation example](/files/docs-example.png)
+[Documentation example](https://zapier.com/engineering/great-documentation-examples/)
+
+For clarity's sake, you can split Documentation testing into Active Testing and Passive Feedback Gathering.
+
+### Passive Feedback Gathering
+Under Passive Feedback Gathering, you can imagine two categories. One is ways to give voice to your developers through Feedbacks, ratings, or more advanced methods. The second part of Passive Feedback Gathering is data mining tools that help you understand developer behavior.
+
+Information gathered via Passive Feedback Gathering helps you make data driven decisions that benefit your readers.
+
+### Active Testing
+Active Testing is testing of your documentation. It is similar to UX testing or Code testing. Unlike the other types of testing, Documentation testing is mainly manual testing. Through manual testing, you can build rapport and empathy with your readers.
+
+## Why You Might Want Documentation Testing?
+Through testing your documentation you understand the first impression of your product. Moreover, you gain insight into the decision funnel and you start to understand who is using your product and why. Documentation testing can help you find weak points in your documentation.
+
+It is a series of good practices and even small teams without tech writers can improve their developer experience by adopting documentation testing.
+
+## Problems Documentation Testing Helps to Solve
+
+* Lack of understanding around your product
+* Lack of developers using your product
+* Damaged information flow
+* Unclear instructions
+* Lack of empathy towards developers
+
+## How to Implement Documentation Testing?
+There are many ways of implementing Documentation Testing. You can start with: 
+### Implementing Passive Feedback Gathering
+Consider the following Passive Feedback Gathering methods:
```

Comment:
This part is weird. There are 2 headlines with one sentence ending with `:` before the next headline. What about to shortly describe all ways in the first paragraph and continue with a detailed description in each paragraph?